dcrrates/server/main.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/decred/dcrdata/v4/dcrrates"
+	"github.com/decred/dcrdata/v4/exchanges"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	killSwitch := make(chan os.Signal, 1)
+	signal.Notify(killSwitch, os.Interrupt)
+
+	cfg, err := loadConfig()
+	if err != nil {
+		fmt.Printf("%v\n", err)
+		return
+	}
+
+	inititializeLogging(filepath.Join(cfg.LogPath, "dcrrate-server.log"))
+
+	if cfg.CertificatePath == "" || cfg.KeyPath == "" {
+		log.Errorf("TLS certificate and key files must be provided")
+		return
+	}
+	creds, err := openRPCKeyPair(cfg.CertificatePath, cfg.KeyPath)
```

Comment:
This may not be straightforward, but a way to specify additional host names for which the cert should be valid would be very helpful.  I had to use `dcrd/cmd/gencerts` with `-H` to make a cert I could use on the rates server host.